source/shared/net/references/transactions.rst

```diff
@@ -450,3 +454,42 @@ If you query the account information of the recipient account afterwards, it wil
 The amount that is not yet released is also included in the ``Balance`` field
 so in this case the account owns ``100 CCD`` that don't belong to any pending
 release schedule.
+
+.. _delegation-commands:
+
+Commands for delegation
+=======================
+
+.. Note::
+
+   Currently delegation is only available on Testnet.
+
+Add delegation
+--------------
+
+The command to add delegation with ``concordium-client`` is
+``delegator add``. For example, an interaction to add delegation on an account looks like the following:
+
+The command is:
+
+.. code-block:: console
+
+   $concordium-client delegator add --capital 5000 --restake 0 --baker 12345
+
+The command has the following optional arguments:
+
+- ``--capital`` is an amount of CCD you intend to delegate
+- ``--restake`` can be set to either 0 or 1 (which is which?) to not restake earnings or restake earnings
+- ``--baker`` is the baker pool ID or L-pool if delegating to the lockup pool.
+
+Configure or change delegation
+------------------------------
+
+The command to add delegation with ``concordium-client`` is
+``delegator configure``. The command has the same arguments as ``delegator add``.
+
+Stop delegation
+---------------
+
+The command to add delegation with ``concordium-client`` is
+``delegator remove``. The command has the same arguments as ``delegator add``?
```

Comment:
Generally, users will specify `--sender`, but the options from `delegator add` are not available.

Reply:
@td202 Can they specify a different sender account to return unstaked funds to than the one that had staked initially?

Reply:
No. The sender account has to be an account with delegation, and the delegation will be removed from that account. No funds are transferred, and it's not possible to transfer the staked funds until the cooldown has expired.